### DISCUSSION TITLE
New component CryptoInput

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#638] [Ideas]
https://app.nostra.finance/stake

<img width="596" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/c6009d2b-45a6-4bb2-8c6c-515ae6fa9856">

https://app.rhino.fi/bridge?token=USDT&chainOut=STARKNET&chain=MATIC_POS

<img width="443" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/1061968/88f7d29c-a06e-4bca-99ee-b3dcd01d2377">

这种代币输入框感觉挺常见的，可以抽象为一个组件，包含如下特性：
- 代币输入
- 展示代币价值（通常是对应多少 $）
- 显示代币 ICon 和名称
- Input 旁边有一些附加信息，比如余额、max（设置为最大可用）等信息

**Comment by yutingzhao1991** [comment]
@jeasonstudio 这个组件感觉可以搞，下周三会上的时候可以讨论下，如果可以的话可以加入到 4 月的设计需求里

**Comment by jeasonstudio** [comment]
按 UI 组件提一版设计需求
